src/clt_least_squares/InputForm.jsx

```diff
@@ -137,7 +137,7 @@ export const InputForm = ({seed, handleSeed, handleGeneratePop, beta,
                         <div className={'form-row'}>
                             <label htmlFor={'variance'}>
                                 <InlineMath>
-                                    {String.raw`\sum^2`}
+                                    {String.raw`\sigma^2`}
```

Comment:
Note: The next field -- "Variance of Disturbances" -- has the input box and tooltip icon squished together. (couldn't add a comment further down)

Reply:
Also, some tooltips end in periods and some do not. Worth a quick review for consistency.